### DISCUSSION TITLE
what each of these parameters means

### DISCUSSION THREAD

**Discussion by clun1** [OpenFAST/openfast#1498] [General]
dear 
hi everyone,I wonder what each of these parameters means?
![image](https://user-images.githubusercontent.com/93849049/226113670-7af05804-5248-4839-a08f-927038b9a962.png)

looking forward your reply

**Comment by jjonkman** [comment]
Dear @clun1,

The InflowWind module's uniform wind file format is documented in the original InflowWind User's Manual: https://openfast.readthedocs.io/en/main/_downloads/a231262e7763b4d61759ebe4e0fc30b2/InflowWind_Manual.pdf.

Best regards,
